CMakeLists.txt

```diff
@@ -250,6 +250,10 @@ option(SWIFT_STDLIB_ENABLE_SIB_TARGETS
        "Should we generate sib targets for the stdlib or not?"
        FALSE)
 
+option(SWIFT_STDLIB_STABLE_ABI
+       "Should stdlib be built with stable ABI (library evolution, resilience)."
+       TRUE)
```

Comment:
Would you mind sinking this into `stdlib/CMakeLists.txt` please?  It's not used by the toolchain at all.